### DISCUSSION TITLE
Training Loss

### DISCUSSION THREAD

**Discussion by seanzlxu** [character-ai/Ovi#67] [Q&A]
Thanks for your wonderful work. I have tried fine-tuning this pretrained model on other languages and found that the loss is very high, especially the audio loss. I also noticed that the loss of the pretrained model on English is also around 2. Is this normal? Would you mind sharing the approximate loss values at each training stage, as well as the final weighted loss of video and audio? Looking forward to your feedback.
